### ISSUE TITLE
Shimgen uses its own executable path instead of the current directory

### ISSUE THREAD

**Issue by alexchandel** [chocolatey/shimgen#22]
When shimgen is invoked from the command line with a relative output, as in calling `C:\ProgramData\chocolatey\tools\shimgen.exe -o="git.exe" --path="/full/path/to/git.exe" ` from `C:\bin`, shimgen places the resulting `git.exe` shim in the `C:\ProgramData\chocolatey\tools\` folder, not the `C:\bin` folder.

**Comment by ferventcoder** [comment]
You mean the `C:\ProgramData\chocolatey\bin` folder right?

**Comment by ferventcoder** [comment]
The output path is relative or absolute. Depends on how it is called.

**Comment by alexchandel** [comment]
Nope, Shimgen puts the executable in `C:\ProgramData\chocolatey\tools\`, right next to itself.

**Comment by alexchandel** [comment]
In other words, it's like the output path is relative to Shimgen's own path, not the current working directory.

**Comment by ferventcoder** [comment]
Ah, I see what you mean.
